README.md

```diff
@@ -1,68 +1,74 @@
-
-The page will reload if you make edits.<br>
-You will also see any lint errors in the console.
-
-### `npm test`
-
-Launches the test runner in the interactive watch mode.<br>
-See the section about [running tests](https://facebook.github.io/create-react-app/docs/running-tests) for more information.
-
-### `npm run build`
-
-Builds the app for production to the `build` folder.<br>
-It correctly bundles React in production mode and optimizes the build for the best performance.
-
-The build is minified and the filenames include the hashes.<br>
-Your app is ready to be deployed!
-
-See the section about [deployment](https://facebook.github.io/create-react-app/docs/deployment) for more information.
-
-### `npm run eject`
-
-**Note: this is a one-way operation. Once you `eject`, you can’t go back!**
-
-If you aren’t satisfied with the build tool and configuration choices, you can `eject` at any time. This command will remove the single build dependency from your project.
-
-Instead, it will copy all the configuration files and the transitive dependencies (Webpack, Babel, ESLint, etc) right into your project so you have full control over them. All of the commands except `eject` will still work, but they will point to the copied scripts so you can tweak them. At this point you’re on your own.
-
-You don’t have to ever use `eject`. The curated feature set is suitable for small and middle deployments, and you shouldn’t feel obligated to use this feature. However we understand that this tool wouldn’t be useful if you couldn’t customize it when you are ready for it.
-
-## Learn More
-
-You can learn more in the [Create React App documentation](https://facebook.github.io/create-react-app/docs/getting-started).
-
-To learn React, check out the [React documentation](https://reactjs.org/).
-
-### Code Splitting
-
-This section has moved here: https://facebook.github.io/create-react-app/docs/code-splitting
-
-### Analyzing the Bundle Size
-
-This section has moved here: https://facebook.github.io/create-react-app/docs/analyzing-the-bundle-size
-
-### Making a Progressive Web App
-
-This section has moved here: https://facebook.github.io/create-react-app/docs/making-a-progressive-web-app
-
-### Advanced Configuration
-
-This section has moved here: https://facebook.github.io/create-react-app/docs/advanced-configuration
-
-### Deployment
-
-This section has moved here: https://facebook.github.io/create-react-app/docs/deployment
-
-### `npm run build` fails to minify
-
-This section has moved here: https://facebook.github.io/create-react-app/docs/troubleshooting#npm-run-build-fails-to-minify
+## Specification
+
+### Views
+
+In Xen Orchestra 6 release, a tree view will be added in order to display objects heicically.
+In order to achieve this, a default map tree view should be set.
+
+#### Basic tree 
+```bash
+pool
+│
+├── VM
+│   ├── VBD
+│   └── VM_snapshot
+├── VM_template
+├── VM_controller
```

Comment:
Nope.

Reply:
Should I delete it ?

Reply:
Nope, but as we said tomorrow in the meeting, it's not at the correct position.

Reply:
Here's a `VM_controller` object. As I see, it's linked to a `pool` and a `VBD`.
Where do you think it's the right position ? 

![Screenshot from 2019-11-19 16-16-10](https://user-images.githubusercontent.com/25125613/69159368-16d57600-0ae8-11ea-8421-10eac3d1e495.png)

Reply:
It's also linked to the host (`$container`).

Reply:
I see now that a `$container` is a `$host`.. So I should move all VM types underneath host, what do you think ?

Reply:
A `$container` is not always a host, it can be a pool if the VM is not on a specific host (not running and disks on shared SR).

Reply:
I see, so putting it under a `host` is a bad idea, I think that we should keep it underneath a `pool`,